tests/e2e/pageobjects/ide/TopMenu.ts

```diff
@@ -45,8 +43,6 @@ export class TopMenu {
         Logger.debug(`TopMenu.clickOnTopMenuButton "${buttonText}"`);
 
         const buttonLocator: By = this.getTopMenuButtonLocator(buttonText);
-
-        await this.ide.closeAllNotifications();
```

Comment:
Could you explain a reason why you deleted this method

Reply:
It makes absolutely no sense to close notifications after clicking the top menu item and it was stealing focus from the `Terminal` element causing the windows to close since the method `closeAllNotifications` is performing clicks.

If you want to make sure there are no notifications present, you should call it in your tests separately before you call `TopMenu.clickOnTopMenuButton`